master/buildbot/status/master.py

```diff
@@ -319,6 +319,13 @@ def getBuilder(self, name):
 
         return None
 
+    def getFriendlyName(self, buildNumber):
+        _builder = self.getBuilder(buildNumber)
+        if _builder is not None:
```

Comment:
I do not like negative logic but it is ok. I prefer:
```
if not _builder:
    return None
return _builder.friendly_name
```

Reply:
Yes, sure. I just did copy-paste to another file.